WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -115,7 +116,7 @@ struct CustomFieldEditorView: View {
             }
         }
         .closeButtonWithDiscardChangesPrompt(hasChanges: viewModel.hasUnsavedChanges,
-                                             closeButtonLabel: { Text(Localization.cancelButton) })
+                                             closeButtonLabel: { Text(isReadOnlyValue ? Localization.doneButton : Localization.cancelButton) })
```

Comment:
On read-only mode, I show "done" instead of "cancel" here, and I hide the other "done" button.